### PR TITLE
Add webcam_interface submodule

### PR DESCRIPTION
**Pull request by emilysxng**
- Add webcam_interface submodule
- Add `opencv v.4.8.1` to the dependency list
  - CXXFLAGS := -std=c++20 -Wall `pkg-config --cflags opencv4`
  - LIBS := `pkg-config --libs opencv4`

Run the following code to run the app:
```
make
./navi
```

`webcam()` is being called in `main.cpp`

### CODE REVIEW

**Comment by AnimeAllstar** [comment]
@CihanBosnali can you get this merged soon so we can add the WebSocket implementation on top of this?

**Comment by AnimeAllstar** [comment]
I do have a question about this. Should this not be a submodule?

**Comment by CihanBosnali** [comment]
Yes, I was about to write that. Please add the folder as a submodule inside /src @emilysxng @droy4405

**Comment by emilysxng** [comment]
I'm having some issues adding it as a submodule since I'm on Windows. My partner Dibbo will update the pull request as he is on Linux, once he gets this working on his laptop. In the meantime I will look into installing WSL.

**Comment by droy4405** [comment]
I will do it once I get home from UBC. Give me few hours.

File: Webcam Interface/CMakeLists.txt
Comment (AnimeAllstar) [inline Webcam Interface/CMakeLists.txt:1]: Do we want to commit this file?

**Review by CihanBosnali — APPROVED** [review]

**Pull request merged**